samples/dnn/text_detection.cpp

```diff
@@ -105,19 +169,16 @@ int main(int argc, char** argv)
         }
 
         // Put efficiency information.
-        std::vector<double> layersTimes;
-        double freq = getTickFrequency() / 1000;
-        double t = net.getPerfProfile(layersTimes) / freq;
-        std::string label = format("Inference time: %.2f ms", t);
+        std::string label = format("Inference time: %.2f ms", average_inference_time);
```

Comment:
looks like there is no averaging of inference time. So maybe just rename to `inferenceTime`?